data/crac/crac-api/src/main/java/com/powsybl/openrao/data/cracapi/Crac.java

```diff
@@ -531,4 +537,71 @@ default State getState(String contingencyId, Instant instant) {
      * Get a {@link RaUsageLimitsAdder}, to add a {@link RaUsageLimits} to the crac
      */
     RaUsageLimitsAdder newRaUsageLimits(String instantName);
+
+    /**
+     * Import CRAC from a file
+     *
+     * @param importers   candidates CRAC importers to process the data
+     * @param inputStream CRAC data
+     * @param cracFactory CRAC factory
+     * @param network     the network on which the CRAC data is based
+     * @return CRAC object
+     */
+    static Crac read(List<Importer> importers, InputStream inputStream, CracFactory cracFactory, Network network) {
```

Comment:
make this private. instead, add a public "read(InputStream inputStream, CracFactory cracFactory, Network network)" method